src/quoting-client.js

```diff
@@ -35,6 +36,7 @@ class QuotingClient {
       let query = {
         source_ledger: source,
         destination_ledger: destination,
+        destination_account: destinationAccount,
```

Comment:
No, its for https://github.com/interledger/five-bells-connector/issues/50 -- convenience, basically. This all works perfectly fine if you don't pass in `destinationAccount` and then fill it in afterwards. Its an optional parameter.